GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -0,0 +1,62 @@
+﻿using CommandLine;
+using GVFS.Common;
+using System;
+
+namespace GVFS.CommandLine
+{
+    [Verb(ConfigVerbName, HelpText = "Get and set GVFS options.")]
+    public class ConfigVerb : GVFSVerb.ForNoEnlistment
+    {
+        private const string ConfigVerbName = "config";
+        private LocalGVFSConfig localConfig;
+
+        [Value(
+                0,
+                Required = true,
+                Default = "",

```

Comment:
Drop these `Default = ""` lines. It's causing the help output to display a confusing `(Default: )` 

```
  GVFS config setting name (pos. 0)     Required. (Default: ) Name of GVFS config setting that is to be set or read

  GVFS config setting value (pos. 1)    (Default: ) Value of GVFS config setting to be set
```